test/data/s3.tfbackend

```diff
@@ -0,0 +1,5 @@
+access_key = "AKIAIOSFODNN7EXAMPLE"
+secret_key = "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
```

Comment:
are these values real?

Reply:
then to avoid trigger false alarm about secret leak, we should add it to `.gitleaks.toml` or change them to dummy values like `access_key = "test"`

Reply:
Good point, I'll just change them to obvious dummy values